src/Database/LSMTree/Internal/Lookup.hs

```diff
@@ -244,19 +237,18 @@ data ByteCountDiscrepancy = ByteCountDiscrepancy {
 --
 -- TODO: don't subdivide into smaller batches here, but inside @blockio-uring@.
 lookupsInBatches ::
-     forall m h. (MonadAsync m, PrimMonad m, MonadThrow m, MonadST m)
+     forall m h. (PrimMonad m, MonadThrow m, MonadST m)
   => HasBlockIO m h
-  -> BatchSize
   -> ResolveSerialisedValue
   -> V.Vector (Run (Handle h))
   -> V.Vector (Bloom SerialisedKey)
   -> V.Vector IndexCompact
   -> V.Vector (Handle h)
   -> V.Vector SerialisedKey
   -> m (V.Vector (Maybe (Entry SerialisedValue (BlobRef (Run (Handle h))))))
-lookupsInBatches !hbio !n !resolveV !rs !blooms !indexes !kopsFiles !ks = assert precondition $ do
+lookupsInBatches !hbio !resolveV !rs !blooms !indexes !kopsFiles !ks = assert precondition $ do
```

Comment:
Do these functions still need to be called "blahInBatches"? So that'd be `lookupsInBatches`, `submitInBatches`.